docs/ci/codemagic.mdx

```diff
@@ -152,14 +152,87 @@ You can find a working example in this repository:
 
   </TabItem>
   <TabItem value="windows" label="Windows" default>
-The documentation for Windows is under development.
-<a href="https://github.com/shorebirdtech/docs/issues/98">
-Contributions welcome</a>.
+
+1. Add the group "shorebird" to your environment groups. If you're using a
+   different group name, use that instead. Also add a variable named `shorebird`
```

Comment:
Codemagic noob question: when/why would I be using "shorebird" as my group name vs a different group name?

Reply:
In Codemagic, you group single secrets into a secret group. Imagine you have the following secrets to upload your iOS app to App Store Connect:

* APP_STORE_CONNECT_PRIVATE_KEY
* APP_STORE_CONNECT_ISSUER_ID
* APP_STORE_CONNECT_KEY_IDENTIFIER 

Now, you bundle them in a group like "ios-publishing". Instead of listing every single secret in the `codemagic.yaml` file, you can just list the group "ios-publishing" and this will add the three secrets.

It's not possible to just a single secret (like in GitHub Actions). A secret always needs a group.

Back to your question: You would use "shorebird" as your group when only have related secrets to Shorebird in this group. However, could also add the `SHOREBIRD_TOKEN` to a different group. In this case, you would need to replace `shorebird` as group. 

Here is an example screenshot from the Codemagic console:

![image](https://github.com/shorebirdtech/docs/assets/24459435/82f876e2-b0a7-40fe-98c4-a93444a59f47)